emcache/_cython/protocol/ascii/multi_line_parser.pxd

```diff
@@ -2,13 +2,12 @@
 # Copyright (c) 2020-2024 Pau Freixes
 
 cdef class AsciiMultiLineParser:
-    cdef:
-        object future
-        bytearray buffer_
-        list keys_
-        list values_
-        list flags_
-        list cas_
-        bytearray client_error_
+    cdef object future
```

Comment:
https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html#c-variable-and-type-definitions